recipes/pcdsutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pcdsutils" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 24e3d8e9c0ad8dd29382b88010a9f42d9db89f5cd0fef1ea14accc517fe27520
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
```

Comment:
You don't need setuptools either. Please just leave pip and python :wink:

Reply:
Thanks.